package.json

```diff
@@ -12,7 +12,8 @@
     "build": "gatsby clean && npm run build:dir && npm run build:favicons && gatsby build --prefix-paths",
     "devbuild": "npm run build:dir && npm run build:favicons && gatsby build",
     "test": "ava **/*.test.js --verbose",
-    "lint": "eslint --max-warnings=0 . --ext ts --ext tsx --ext js --ext jsx"
+    "lint": "eslint --max-warnings=0 . --ext ts --ext tsx --ext js --ext jsx",
+    "lintFix": "eslint --fix . --ext ts --ext tsx --ext js --ext jsx"
```

Comment:
Was lintFix introduced on purpose or by mistake?

Reply:
I added it on dev for me to quickly fix some lint warning, I wil remove it

Reply:
Done